src/DynamoPublish/Models/PublishModel.cs

```diff
@@ -227,7 +229,8 @@ static PublishModel()
             managerURL = appSettings.Settings["ManagerPage"].Value;
         }
 
-        internal PublishModel(IAuthProvider dynamoAuthenticationProvider, ICustomNodeManager dynamoCustomNodeManager)
+        internal PublishModel(IAuthProvider dynamoAuthenticationProvider, 
+            ICustomNodeManager dynamoCustomNodeManager, CameraData dynamoCameraData)
```

Comment:
when send is being invoked it can't retrieve camera data because neither PublishModel nor PublishViewModel know about DynamoPublishExtension which can get the camera info from its loadedParams. On the other hand, DynamoPublishExtension doesn't know when send is invoked to pass camera info as a parameter to corresponding method.
So I don't see a way to implement sending camera info without passing it to constructor..
Please correct me if I missed or didn't get something